android/app/proguard.cfg

```diff
@@ -0,0 +1,32 @@
+##---------------Begin: proguard configuration for Gson  ----------
```

Comment:
do you think this is needed?
AFAIK we don't use proguard (we do use r8 which came default with flutter)